axelar-chains-config/info/testnet.json

```
@@ -1,5 +1,15 @@
 {
   "chains": {
+    "centrifuge": {
+      "name": "centrifuge",
+      "id": "centrifuge",
```

```suggestion
      "id": "centrifuge-2",
```
since it was added before already